examples/nestjs/consumer/src/app.service.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpService, Injectable } from '@nestjs/common';
+import { Animal } from './animal.interface';
+
+@Injectable()
+export class AppService {
+  private static readonly AUTH_HEADER = { authorization: 'Bearer token' };
+
+  public constructor(private readonly http: HttpService) {}
+
+  private static getApiEndpoint() {
+    return process.env.API_HOST || 'http://localhost:8081';
```

Comment:
We've spoken about this already, but I think this would be better injected idiomatically, rather than passed through environment variables (apologies if this is done already and I missed the branch or something)